packages/fx-core/src/component/generator/apiSpec/generator.ts

```diff
@@ -127,7 +127,7 @@ export class SpecGenerator extends DefaultTemplateGenerator {
     let templateName = "";
     if (
       (capability === CapabilityOptions.apiPlugin().id ||
```

Comment:
Can CapabilityOptions.apiPlugin() be deleted?

Reply:
Yes, I will create an another task to do this clean up. The current UI update is a little bit urgent.